sktime/forecasting/base/adapters/_generalised_statsforecast.py

```diff
@@ -26,7 +29,16 @@ class _GeneralisedStatsForecastAdapter(BaseForecaster):
     def __init__(self):
         super().__init__()
 
-        self._forecaster = None
+        self._forecaster = self._instantiate_model()
+        (
+            self._support_pred_int_in_sample,
+            self._support_pred_int,
+        ) = self._check_supports_pred_int()
+
+        if not self._support_pred_int_in_sample:
+            self.set_tags(**{"capability:pred_int:insample": False})
+        if not self._support_pred_int:
+            self.set_tags(**{"capability:pred_int": False})
```

Comment:
As of now, these tags are being passed in concrete classes. If you handle these here, totally fine, but may be we can remove from there? And we can always (both True/False cases) set tags as `self.set_tags(**{"capability:pred_int:insample": self._support_pred_int_in_sample})` or similarly I guess.